NEWS.md

```diff
@@ -1,60 +1,55 @@
-3.23.1
+3.24.1
 ======
 
+## Highlights
 
-## Features
-
- * The `redis()` destination now handles any number of command parameters.
-   (#2816)
-
- * The `format()` option of `date-parser()` supports lists. From now on, a
-   single `date-parser()` instance is able to process different date formats,
-   making it easy to catch on when some programs change the way they log
-   information. (#2779)
+ * Add a new template function called `format-flat-json()`, which generates
+   flattened json output. This is useful for destinations, where the json
+   parsing does not handle nested json format. (#2890)
+ * Add ISO 8601 compliant week numbering. Use it with the `${ISOWEEK}` macro
+   and the timestamp prefixes `P_, S_, R_, C_` (#2878)
+ * Send Server Name Identification (SNI) information with `transport(tls)`.
+   Enable it by setting the `sli(yes)` option in the `tls` block in your
+   `destination`. (#2930)
 
```

Comment:
Don't forget to add the `add-contextual-data` glob feature here (I know that the PR is not merged yet, but it will be part of 3.24).

Reply:
Thanks, I added it! :)